i18n/en.json

```diff
@@ -5607,6 +5607,10 @@
     "id": "app.job.download_export_results_not_enabled",
     "translation": "DownloadExportResults in config.json is false. Please set this to true to download the results of this job."
   },
+  {
+    "id": "app.job.error",
+    "translation": "Error during job execution."
```

Comment:
Is there anything the user who sees this message can do as a workaround? If so, do we want to add those workaround details to the error message?

Reply:
This is just a generic message for when a Job fails. What the job is doing exactly is unknown at this point. The error message presented to the user (for example the admin importing a backup into mattermost) consists of this message and the error reported by the actual job.